scripts/test-helpers

```diff
@@ -492,35 +492,57 @@ provision-registry-proxy() {
     local prefix="docker-registry-"
     local registries="docker.io:registry-1.docker.io k8s.gcr.io gcr.io quay.io ghcr.io"
     local registries_yaml="$TEST_DIR/registries.yaml"
-
-    echo "mirrors:" > $registries_yaml
-
-    for registry in $registries; do
-        IFS=: read registry_name registry_endpoint <<< $registry
-        if [ -z "$registry_endpoint" ]; then
-            registry_endpoint=$registry_name
-        fi
-
-        local name="registry_${registry_name//./_}"
-        local status=$(docker inspect $name --format '{{ .State.Status }} {{ .Config.Image }} {{ (index .HostConfig.PortBindings "5000/tcp" 0).HostPort }}' 2>/dev/null || true)
-
-        read state_status config_image hostport <<< $status
-        if [ "$state_status" != "running" ] || [ "$config_image" != "$image" ]; then
-            hostport=$(timeout --foreground 5s bash -c get-port)
-            docker rm --force $name 2>/dev/null || true
-            docker run \
-                -d --name $name \
-                -p 0.0.0.0:$hostport:5000 \
-                -v "registry-cache:/var/lib/registry" \
-                -e "REGISTRY_HTTP_SECRET=shared-secret" \
-                -e "REGISTRY_PROXY_REMOTEURL=https://$registry_endpoint" \
-                -e "REGISTRY_STORAGE_CACHE_BLOBDESCRIPTOR=inmemory" \
-                -e "REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY=/var/lib/registry/$registry_name" \
-                $image
-        fi
-        echo -e "  $registry_name:\n    endpoint:\n    - http://172.17.0.1:$hostport" >> $registries_yaml
-    done
-
+    local seeder_registry="[https://seeder.int.rancher.io:5000/]"
```

Comment:
Rather than embedding a specific registry configuration that is automatically enabled by introspecting the local docker instance, could we perhaps host the registry configuration somewhere on the seeder, and point at that with an environment variable? That way we only need to support a single variable (REGISTRIES_CONFIG_URL or something) that would be curl'd and used as the test's registries.yaml. This would also let you make modifications or take the mirrors down for maintenance without breaking our CI or needing to put in a PR.